profiling/README.md

```diff
@@ -0,0 +1,26 @@
+## Profiling for Node  
+- CPU Profiling
+  - V8 CPU Profiler: [v8/include/v8-profiler.h](https://github.com/v8/blob/master/include/v8-profiler.h)
+    - Accessible globally through `--prof` runtime flag, basic processing and display through `--prof-processor` runtime flag
+  - Intel vTune profiler. Accessible through `--enable-vtune-profiling` compile-time flag.
+
+- Heap Profiling
+  - V8 Heap Profiler: [v8/include/v8-profiler.h](https://github.com/v8/blob/master/include/v8-profiler.h)
+    - Also accessible through built-in 'v8' module.
```

Comment:
The v8 module provides access to heap statistics, that's not quite the same thing as the heap profiler.